app/src/components/map.jsx

```diff
@@ -211,28 +217,6 @@ class Map extends Component {
   }
 
   /**
-   * Draws a line that connects by timestamp order all paths of the following vessel
```

Comment:
AFAIK this is dead code, it was replaced when we added the vessel tracks using the detailed data